### DISCUSSION TITLE
Wow! Monumental achievement.

### DISCUSSION THREAD

**Discussion by sandyol55** [signag/raspi-cam-srv#1] [Ideas]
Found this repo based on your link provided in the Issues on Picamera2.
Have cloned and experimented with a small subset of the features on a Pi5 with an IMX219 V2 camera. Everything tried so far works great.

If you are accepting any feature requests I'd like to vote for  a form of motion-triggered video recording in addition to the remote control from the client browser.

As a 'non-programmer'  I've had a go at stitching together some of the Picamera2 examples to give a stream and motion-triggered video capture package but it is VERY basic when compared to raspi-cam-srv.

Your package seems to be set to be a worthy successor to the venerable RPi-Cam-Web Interface , look forward to seeing how it progresses.

**Comment by signag** [comment]
Glad to hear that this worked for you.
Indeed, the next steps would be to add useful functionality, especially since the camera system does not allow concurrent access to the camera while the server has a hold on it.

My first step will be Timelapse, since this was actually my trigger to deal with this.
Motion triggered video might be a candidate, too.

**Comment by signag** [comment]
> I'd like to vote for a form of motion-triggered video recording

This feature is currently available in a basic version and came with V2.2.0

**Reply by sandyol55** [reply]
Yes,I've cloned v2.2.2 and been testing the motion triggering. Everything I've tried so far  has been working as described.
Thanks for your efforts to bring this forward.
